src/css/styles.scss

```diff
@@ -187,3 +188,31 @@ ul {
   color: white;
 }
 
+.grid-item:hover {
+  border-color: hsla(0, 0%, 100%, 0.5);
+  cursor: move;
+}
+
+.friend-icon-container {
+  height: 100%;
+  background: #ffffff;
+  padding: 5% 5%;
+  width: 100%;
+}
+
+.friend-button {
+  background-color: #039BE5;
```

Comment:
would it have been possible to use the scss color variables we have for this? and in other places within this?

Reply:
I use scss when it's more then a random one off. It's not needed for this.